vmdb/app/models/miq_expression.rb

```diff
@@ -31,6 +33,7 @@ class MiqExpression
     OntapStorageVolume
     OntapVolumeMetricsRollup
     PolicyEvent
+    Provider
```

Comment:
The base model for ExtManagementSystem is already in the dictionary (/vmdb/config/locales/en.yml) as "Provider" so this entry creates a duplicate name in the drop-down list for Models to base the report on.  I think we may want to expose the ProviderForeman model here instead of the base class.

However, the ProviderForeman model likely is not exposing the data we want to report on.  That data comes from the ConfigurationManager and ProvisioningManager models which would need to be exposed here.